pyproject.toml

```
@@ -29,7 +29,7 @@ google-api-core = "^2.11.0"
 protobuf = ">=3.20,<5.0"
 aiorun = "^2023.7"
 uvloop = "^0.19.0"
-psutil = "^5.9.8"
+psutil = ">=5.9.8"
```

```suggestion
psutil = "^6.0"
```
But where are we using this in code?

---

https://github.com/numaproj/numaflow-python/blob/1f839005290eaa0aef870c6973d8d16a14223977/pynumaflow/shared/server.py#L284